examples/playground/cypress/integration/table/ActionableRows_spec.js

```diff
@@ -0,0 +1,22 @@
+describe("Actionable Rows", function () {
+	beforeEach(function () {
+		cy.visit("http://localhost:21356/#/components/table");
+	});
+
+	context("Navigate to Actionable Rows Table", function () {
+		it("can navigate to 'actionable rows' from tab options", function () {
+			cy.get(".btn-group").within(function () {
+				cy.contains("actionable rows").click();
+			});
+			cy.get(".table-responsive").within(function () {
+				cy.get(".table-row")
+				.first()
+				.should("contain", "Darth Vader");
```

Comment:
When breaking up chained functions over multiple lines, please indent the line-breaks by one line from the parent.

```js
cy.get(".table-row")
    .first()
    .should()
```

for example.